docs/source/specs/openapi.json

```diff
@@ -613,7 +613,7 @@
             "content": {
               "application/json": {
                 "schema": {
-                  "$ref": "#/components/schemas/GroupWithPrincipalsAndRoles"
+                  "$ref": "#/components/schemas/GroupRolesPagination"
```

Comment:
I think this will just have `RoleOut`, not `GroupRolesPagination`, won't it?

Reply:
If we change it to RoleOut, let me know. For the sake of charging forward with test automation I used GroupRolesPagination when I generated my most recent client source.

Reply:
Yeah, I don't think we're returning the paginated schema for a POST response. We're just returning the `RoleOut` schema, unless I'm mistaken.

Reply:
Right, it will return a list of RoleOut. Just updated 👍